### DISCUSSION TITLE
Queries regarding latest update

### DISCUSSION THREAD

**Discussion by michallawson** [HCL-TECH-SOFTWARE/connections-automation#109] [Q&A]
I realise this is probably a fairly basic question, but with the new quick start you have used a dynamic host for the cnx application ingress. When setting up the dns should this have the same IP address as web.internal,example.com? or should it be the same as connections.internal.example.com?

I see on Flexnet that a new version (2104) of the update wizard has been released. Can I just change the file name in all.yml to the new version and use that to install the ifix?

Many thanks

Michael

**Comment by dejo1307** [comment]
Hey Michael, 

Sorry for waiting as I am currently out of office. 

Dynamic host is whatever is your entry point; in quick start context it would be the IP of web.internal.connections.com.

Regarding new version of update wizard - yes, you should just be able to change that one. As I took some time off I didn't notice that there is newer one :)

Best regards,
Dejan

**Reply by michallawson** [reply]
Dejan,

Thank you.
